scripts/jiva-tools/jiva_preupgrade_checks.sh

```diff
@@ -62,12 +62,12 @@ echo "Total used size: $size"
 
 echo "ChainLen:$chainLen"
 
-find . -iname '*.img'
+#find . -iname '*.img'
 imgCnt=$(find . -iname '*.img' | wc -l)
 
 echo "img files count:$imgCnt"
 
-find . -iname '*.img.meta'
+#find . -iname '*.img.meta'
 metaCnt=$(find . -iname '*.img.meta' | wc -l)
 
 echo "meta files count:$metaCnt"
```

Comment:
The error messages below can say: 
- Chain Length does not match with available img files
- Chain Length does not match with available meta files